pyccel/codegen/printing/cucode.py

```diff
@@ -61,14 +61,45 @@ def _print_Module(self, expr):
             else:
                 local_imports += self._print(imp)
 
-        imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
-                    {local_imports}\
-                    }}'
+        imports = f'{c_headers_imports}\n\
+                    {local_imports}\n'
```

Comment:
This code is fine but you may be interested to know that there is another way to do this:
```suggestion
        imports = (f'{c_headers_imports}\n'
                   f'{local_imports}\n')
```

Reply:
fixed 3580b26